dot/telemetry/mailer_test.go

```diff
@@ -95,23 +95,34 @@ func TestHandler_SendMulti(t *testing.T) {
 		),
 
 		NewAfgAuthoritySet("authority_id", "authority_set_id", "json-stringified-ids-of-authorities"),
-		NewAfgFinalizedBlocksUpTo(
-			common.MustHexToHash("0x07b749b6e20fd5f1159153a2e790235018621dd06072a62bcd25e8576f6ff5e6"), "1"),
-		NewAfgReceivedCommit(
-			common.MustHexToHash("0x5814aec3e28527f81f65841e034872f3a30337cf6c33b2d258bba6071e37e27c"),
+
+		func(hash common.Hash, number string) Message {
+			return NewAfgFinalizedBlocksUpTo(hash, number)
+		}(common.MustHexToHash("0x07b749b6e20fd5f1159153a2e790235018621dd06072a62bcd25e8576f6ff5e6"), "1"),
```

Comment:
Cant't we define an implementation of `Message` directly instead of this convoluted wrapping?